opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -73,6 +96,28 @@ namespace Opm {
         return fun::map( mknode, wnames );
     }
 
+    static inline std::vector< ERT::smspec_node > keywordW(
```

Comment:
Some refactoring suggestions:
1. rename this `defaultW( )`.
2. Remove the `missing` lambda and the `wnames.erase( )` - they never kick in.
3. The default, i.e. have-no-data code path in the other `keywordW( )` method can be implemented in terms of this new `defaultW( )` function.
4. Repeat for `keywordG( )`.